### DISCUSSION TITLE
Will the latest version work on 1.21?

### DISCUSSION THREAD

**Discussion by DeadGuitar** [Athar42/CustomDiscs#2] [Q&A] [answered]
I just wonder if I can use custom goat horns on my 1.21 server

**Comment by Athar42** [comment, marked as answer]
Hi,

The release (3.2.1a) is working on 1.21/1.21.1 PaperMC release and support customized Goat Horns.

A newer release (going online in a few seconds) will support 1.21.3 (but take note of the warning in the release note :D ).

See https://github.com/Athar42/CustomDiscs/releases for all the versions availables :)
